django-s3-file-field-client/src/client.ts

```diff
@@ -25,10 +25,19 @@ export interface UploadResult {
   state: 'aborted' | 'successful' | 'error';
 }
 
-export default class S3FFClient {
-  protected readonly baseUrl: string;
+export interface ProgressEvent {
+  loaded?: number;
```

Comment:
```suggestion
  uploaded?: number;
```

Seems like a more sensible name.